packages/components/src/modal/test/index.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * External dependencies
+ */
+import { render, screen, within } from '@testing-library/react';
```

Comment:
Nice! I didn't know about this `within` method. 👍

Reply:
Neither did I! Required a bit of googling for how to test portals.